### ISSUE TITLE
NPE on startup

### ISSUE THREAD

**Issue by zerafox** [Rocologo/MobHunting#161]
1.11.2 Spigot
4.0.0 MythicMobs
2.3.5 MyPet (Premium)

Incase any of that matters. I've seen reports of conflicts with both recently.

```18.02 21:05:04 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_111] 
18.02 21:05:04 [Server] INFO at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_111] 
18.02 21:05:04 [Server] INFO at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_111] 
18.02 21:05:04 [Server] INFO at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) [custom18.jar:git-Spigot-b4417b8-3c48711]
 18.02 21:05:04 [Server] INFO at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[custom18.jar:git-Spigot-b4417b8-3c48711] 
18.02 21:05:04 [Server] INFO at one.lindegaard.MobHunting.compatibility.CitizensCompat$1.run(CitizensCompat.java:80) ~[?:?] 
18.02 21:05:04 [Server] INFO java.lang.NullPointerException 18.02 21:05:04 [Server] WARN Plugin MobHunting v4.4.8 generated an exception while executing task 219```

**Comment by Rocologo** [comment]
This one looks like somthing whit citizens and not mypet/mythicmobs. Which version of citizens do you use?

**Comment by Rocologo** [comment]
You need to tell me more if I am going to do someting about this  ticket.

**Comment by zerafox** [comment]
Without scouring the error, and being away from my PC and being unable to check it until now....

2.0.21 SNAPSHOT build 1463 CITIZENS

Outside of a 'citizens build #' what else can I tell you more on.. -.-

**Comment by Rocologo** [comment]
Try build 295 to see if this removes the error.

http://jenkins.lindegaard.one:8180/job/MobHunting/

**Comment by Rocologo** [comment]
Fixed in v4.5.4
